roles/common/files/sensu_plugins/check-hostname.sh

```diff
@@ -1,24 +1,23 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Is that bit really necessary? Do we ever rely on a bash that's not in /bin/bash ?